styles/navigation/navbar/navbar.twig

```diff
@@ -0,0 +1,9 @@
+<ul class="navbar">
+  {%for item in items%}
+    <li class="navbar__item">
+      <a href="{{ item.url }}" class="navbar__link{% if item.active %} is-active{% endif %}">
```

Comment:
1) is-active on tarkotus olla yleispätevä monessa paikassa käytettävä? Sen sijaan että ois navbar__link--active tms?

2) inline iffit on hivenen hämääviä, ois ehkä jonkin verran selkeempi jos ois joku classes muuttuja ja sinne iffiteltäis lisää classeja